config/nightwatch.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
   custom_commands_path: './test/util/nightwatch-commands',
   live_output: true,
   parallel_process_delay: 10,
-  disable_colors: false,
+  disable_colors: true,
```

Comment:
No, that's definitely possible. I didn't do that because I was going to ask if people were attached to color output locally.